.eslintrc.js

```diff
@@ -2,85 +2,85 @@
 const jestVersion = require('jest/package.json').version
 
 module.exports = {
-  extends: [
-    'eslint:recommended',
-    'next/core-web-vitals',
-    'plugin:@typescript-eslint/recommended',
-    'prettier',
-  ],
-  plugins: ['unicorn'],
-  settings: {
-    'import/internal-regex': '^(@app-|~)',
-    jest: {
-      version: jestVersion,
-    },
-  },
-  rules: {
-    // Base
-    'no-shadow': 'error',
-    'no-warning-comments': 'off',
+	extends: [
```

Comment:
I'm okay with this idea. But 4 spaces per tab is too wide for me.

Can you make it so it uses 2 spaces per tab?

Reply:
That's one of the advantages of tabs: each user decide the size they want to see for the tabs!

I can't set that, that's up to you in your IDE. As I mentioned in the description, you can customize it for GitHub here: https://github.com/settings/appearance

Reply:
> That's one of the advantages of tabs: each user decide the size they want to see for the tabs!

Oh that's the big idea here!

I forgot to tell GitHub how many spaces it should use for tabs. I've changed my settings on GitHub, and it looks the same as before now. 🥳 

Yeah, let's try this out. In any case it will make merging a Prettier `3.0.0` PR a bit easier, as the files are already formatted in the correct way. 😄